packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -126,6 +125,15 @@ export default class Generate extends Command {
   };
 
   cleanFiles = (appName: string): void => {
-    fs.rmdirSync(`${appName}/.add-ons`)
+    cli.info('Removing the .add-ons folder.')
+    fs.rmSync(`${appName}/.add-ons`, {recursive: true})
+  };
+
+  displayEndMessage = (appName: string): void => {
+    cli.info(``)
+    cli.info(`\n\n🚀 Your app "${appName}" has been created successfully! 🥳`)
```

Comment:
Do we need emoji here? 😬

Reply:
**Opinionated answer:** YES! (just kidding)
**Objective answer:** While it helps attract the attention of the users and acts as a "TL;DR" thing, I agree that we do not absolutely need it :D

Reply:
**Updated:** I removed the party and bulb icons to keep only the rocket, does it sound better? :D

Reply:
Yep, looks good 🙂 Thanks!